### ISSUE TITLE
Pages or Groups sometimes jump between browser windows

### ISSUE THREAD

**Issue by Loevbo** [loilo/auto-group-tabs#44]
If i have multiple browser windows open, example 2 chrome browser next to each other. Then sometimes special after load of a page, then the page jumps to the other browser.

Lets say I have browser 1 and browser 2

On browser 1 i have chatGPT open. I now write a question and hit enter, and GPT now generate the answer.
While GPT is writing the answer i go to Browser 2. While im on Browser 2 then GPT is done and some times GPT will jump to browser 2.

**Comment by Hyug** [comment]
Hi there, thanks @loilo for this add-on, i was dreaming of it for a while !
I would add an enhancement to this issue : I'm using several browser windows too, like "environments" (team work, personal work, personal research and so on...). As you may know, when opening a link from another source (mail client, chat client, etc) the link will open in the last used browser window, and Auto-group Tabs is not grouping the newly opened tab to the potentially existing group in another browser window, but it creates a new group in the current browser window.
May be you can add a setup, general or by group to activate or not multi-windows grouping !
Thank you !

**Comment by loilo** [comment]
Hey @Hyug,
you can do this on a per-group base. Click the "Advanced Settings" button and tick the "Merge" checkbox. Then, an existing group in another browser window will be preferred to creating a new one in the current window.

That said, as a general note: Please do not chime into someone else's issue unless you're like 99% sure it's actually the same problem. It makes separating problem spaces harder for me – just create a new issue in such a case, they're free. :)

**Comment by Loevbo** [comment]
@loilo Any update on the problem i descriped? I'm not having the same problem is @Hyug

**Comment by loilo** [comment]
@Loevbo Not really, I didn't really have time to dig deeper into the extension since you opened the issue, sorry.

**Comment by Hyug** [comment]
Hi @loilo thanks for your reply and the merge option, Sorry for issueceptionning...
